src/docs/src/whatsnew/3.5.rst

```diff
@@ -77,17 +77,48 @@ Highlights
 * :ghissue:`5347`: Fix attachment size calculation. This could lead to shards
   not being scheduled for compaction correctly.
 
+* :ghissue:`5494`: Implement ``_top_N`` and ``_bottom_N`` reducers. These
+  reducers return the top or bottom values from a map-reduce view. ``N`` is a
+  number between ``1`` and ``100``. For instance, a ``_top_5`` reducer will
+  return the top 5 highest values for a group level.
+
+* :ghissue:`5498`: Implement ``_first`` and ``_last`` reducers. These reducers
+  return the first, and respectively, the last view row for a given group
+  level. For example, ``_last`` can be used to retrieve the last timestamp
+  update for each ``device_id`` if we had a key like ``[device_id, timestamp]``
+  if we query the view with ``group_level=1``.
+
+* :ghissue:`5466`: Conflict finder plugin. Enable the
+  ``couch_scanner_plugin_conflict_finder`` plugin to find conflicting docs
+  across all the databases. It can be configured to report individual revisions
+  or aggregated statistics.
+
 Performance
 -----------
 
+* :ghissue:`5499`: QuickJS rope based string implementation.
+* :ghissue:`5451`: Optimize config system to use persistent terms.
 * :ghissue:`5437`: Fix ``atts_since`` functionality for document ``GET``
   requests. Avoids re-replicating attachment bodies on doc updates.
-* :ghissue:`5389`: Save 1 ``write`` for each committing data to disk by using
+* :ghissue:`5398`: Save 1 ``write`` for each committing data to disk by using
   ``fdatasync`` while keeping the same level of storage reliability.
 
 Features
 --------
 
+* :ghissue:`5526`: Default ``upgrade_hash_on_auth`` to ``true``. Downgrading to
+  ``3.4.1``, ``3.4.2``, or ``3.4.3`` is safe. Those versions know how to verify
+  these new password hashes.
+* :ghissue:`5517`: Enable xxHash file checksums by default. Downgrading to
+  ``3.4.x`` versions should be safe. Those versions know how to read and verify
```

Comment:
@nickva small nit: at the previous point we use “it is safe to …” and here “it should be safe…”. We should use the same wording.

Reply:
Thanks!